rotkehlchen/chain/ethereum/modules/uniswap/v2/common.py

```diff
@@ -352,7 +367,7 @@ def enrich_uniswap_v2_like_lp_tokens_transfers(
     """This function enriches LP tokens transfers of Uniswap V2 like AMMs."""
     resolved_asset = context.event.asset.resolve_to_crypto_asset()
     if (
-        resolved_asset.symbol == lp_token_symbol and
+        resolved_asset.symbol.startswith(lp_token_symbol) and
```

Comment:
Why startswith here? Due to the `object.__setattr__(pool_token, 'symbol', f'{pool_token.symbol} {token0.symbol}-{token1.symbol}')` that the way you have it now happens only at 2nd run?

Reply:
yes